packages/webapp/src/pages/Connection/Show.tsx

```diff
@@ -265,7 +266,12 @@ We could not retrieve and/or refresh your access token due to the following erro
                             <ErrorCircle />
                             <span className="ml-2">There was an error refreshing the credentials</span>
                             <Link
-                                to={`/${env}/activity?activity_log_id=${connectionResponse.errorLog.activity_log_id}&connection=${connectionResponse.connection.connection_id}&date=${getSimpleDate(connectionResponse.errorLog?.created_at?.toString())}`}
+                                to={getLogsUrl({
```

Comment:
When clicking from the connections page I get this

![image](https://github.com/NangoHQ/nango/assets/1724137/80ac3ae9-db57-4f3a-a7b4-f00c219a3558)

which in of itself the x looks misaligned

Reply:
https://github.com/NangoHQ/nango/assets/1724137/a503dee2-ba0c-44df-a0b4-2763a2c66735

Reply:
weird, thanks for testing that looking into it

Reply:
I don't have any issue on my side, however I noted the Failed state does not appear if the schedule is paused